### ISSUE TITLE
bdr cannot be pg_upgrade'd due to use of regprocedure columns

### ISSUE THREAD

**Issue by ringerc** [2ndQuadrant/bdr#79]
The BDR catalogs use a `regprocedure` column in `bdr.bdr_conflict_handlers.ch_fun`. Because user proc oids aren't stable across `pg_upgrade`, it does the right thing and refuses to upgrade with, e.g.:

```
Checking for reg* system OID user data types                fatal

Your installation contains one of the reg* data types in user tables.
These data types reference system OIDs that are not preserved by
pg_upgrade, so this cluster cannot currently be upgraded.  You can
remove the problem tables and restart the upgrade.  A list of the problem
columns is in the file:
    tables_using_reg.txt
Database: test_db
  bdr.bdr_conflict_handlers.ch_fun
  bdr.bdr_list_conflict_handlers.ch_fun
```

(Report per @amenonsen, seen while testing for a 0.7.x to 0.9.1 upgrade that brings a server from 9.4beta2+bdr to 9.4.2+bdr)

**Comment by ringerc** [comment]
It seems clear that we should be referring to procedures by their schema and name here, even if it's a bit less convenient.

**Comment by ringerc** [comment]
It looks like @PJMODOS already fixed this in bdr-plugin/next, see 6e3a725d . It probably isn't wise to backport this, so closing as resolved.
